### PR TITLE
Create rxJava.travis.yml

### PR DESCRIPTION
**Pull request by genisysram**

### CODE REVIEW

**Review by gururajrkatti — CHANGES_REQUESTED** [review]
command in the script doesn't seem to be doing anything. Pl. check

**Comment by genisysram** [comment]
@gururajrkatti 
Modified the code.

File: travis-ymls/rxJava.travis.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - amd64
+    - ppc64le
+
+install: true
+
+sudo: required
+
+# script for build and release via Travis to Bintray
+script: 
+    - ./gradlew
```
Comment (gururajrkatti) [inline travis-ymls/rxJava.travis.yml:31]: You are still using just this command. I think it's not doing anything as per logs.

**Comment by genisysram** [comment]
@gururajrkatti 
Modified the code

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**